control/veda_heal.py

```diff
@@ -149,7 +149,7 @@ def determine_fault(self, video_object):
         check_list = []
         if encode_list is not None:
             for e in encode_list:
-                if e != 'mobile_high' and e != 'audio_mp3' and e != 'review' and e != 'hls':
+                if e != 'mobile_high' and e != 'audio_mp3' and e != 'review':
```

Comment:
@muzaffaryousaf These encodes can be incomplete and the video can still be read as "complete". They're secondary to users viewing. (HLS was secondary, but now it's part of the main process).